CodeGeneration/Sources/SyntaxSupport/AttributeNodes.swift

```diff
@@ -168,8 +168,8 @@ public let ATTRIBUTE_NODES: [Node] = [
         documentation: "The colon separating the label and the value"
       ),
       Child(
-        name: "AvailabilityList",
-        kind: .collection(kind: .availabilitySpecList, collectionElementName: "Availability")
+        name: "AvailabilitySpecs",
+        kind: .collection(kind: .availabilitySpecList, collectionElementName: "AvailabilityArgument")
```

Comment:
`name` and `collectionElementName` should match here. Also, I’m not a huge fan of the `Spec` part. IMO it would be nice to also rename `AvailabilitySpecListSyntax` -> `AvailabilityArgumentListSyntax`. https://github.com/apple/swift-syntax/issues/1737

---

So, I think we should use name: `AvailabilityArguments`.

--- 

And, could you also rename the `.availabilitySpecList` argument of `.availabilityCondition` to be consistent with this one?